Source/Client.swift

```diff
@@ -61,7 +61,7 @@ internal final class Client: ClientType {
         let request = urlRequest(url: Service.postInstall.url, parameters: parameters)
         enqueueRequest(request: request, completion: { data, _ in
             guard let data = data,
-                let responseDict = try? JSONSerialization.jsonObject(with: data) as? [String: Any],
+                let responseDict = ((try? JSONSerialization.jsonObject(with: data) as? [String: Any]) as [String: Any]??),
```

Comment:
With the update to Swift 5, I think we can do this, which should fix the warning
```
let object = responseDict["object"] as? [String: Any],
```